Sources/MapboxNavigation/BannerDismissalViewController.swift

```diff
@@ -9,6 +9,16 @@ public class BannerDismissalViewController: UIViewController, Banner {
     
     var backButton: BackButton!
     
+    // :nodoc:
+    public var backTitle: String? {
```

Comment:
Thanks! Can you please also add unit-test for that? And for default case as well (when default title is used)?

Reply:
Thanks for the suggestion, the unit test is added.